.coveragerc

```diff
@@ -2,13 +2,15 @@
 branch = True
 omit =
     */tests*
+    */test*
```

Comment:
Yeah, we need both. Essentially this file is saying "don't include coverage information for these directories in the total summary"—it was already excluding the `tests/` directory (i.e. `labcontrol/db/tests/`, but it wasn't excluding the `test/` directory for the front-end (`labcontrol/gui/test/`). This change should fix that problem, and the result is that coveralls should now provide a more realistic representation of the code coverage.

...It might also be worth renaming either of the test directories to make them consistent, since having the test directories be named slightly differently is a little bit confusing :)

Reply:
> ...It might also be worth renaming either of the test directories to make them consistent, since having the test directories be named slightly differently is a little bit confusing :)

I agree, opening an issue.

Reply:
Marking this as resolved—we can follow up with this in #589.